src/population.cpp

```diff
@@ -142,6 +142,10 @@ uint64_t Population::size() const {
     return dset.getSpace().getDimensions()[0];
 }
 
+Selection Population::selectAll() const {
```

Comment:
One more empty line before?

Also, couldn't this be inlined?

Reply:
Good call, and I suppose (although I trust the compiler :))

Reply:
Hmm, on second thought, none of the other methods are inlined yet, so I'd prefer to do that in another PR, and not split one out currently.

Reply:
I think just the non-`impl_` concerning methods should be inlined, right?

Reply:
Would be worth checking if GCC w/ LTO is already doing that, just for fun.

I don't think it's something that would be rejected if someone makes a PR, assuming it isn't super intrusive.  I just don't have time to do it now.